build/cloud/01-prepare.yaml

```diff
@@ -0,0 +1,608 @@
+                      description: PrivateKey for access opc server.
+                      type: string
+                    securityMode:
+                      description: Defaults to "none".
+                      type: string
+                    securityPolicy:
+                      description: Defaults to "none".
+                      type: string
+                    timeout:
+                      description: Timeout seconds for the opc server connection.???
+                      format: int64
+                      type: integer
+                    url:
+                      description: 'Required: The URL for opc server endpoint.'
+                      type: string
+                    userName:
+                      description: Username for access opc server.
+                      type: string
+                  required:
+                    - url
+                  type: object
+              type: object
+          required:
+            - deviceModelRef
+          type: object
+        status:
+          properties:
+            twins:
+              description: A list of device twins containing desired/reported desired/reported
+                values of twin properties. A passive device won't have twin properties
+                and this list could be empty.
+              items:
+                properties:
+                  desired:
+                    description: 'Required: the desired property value.'
+                    properties:
+                      metadata:
+                        description: Additional metadata like timestamp when the value
+                          was reported etc.
+                        type: object
+                      value:
+                        description: 'Required: The value for this property.'
+                        type: string
+                    required:
+                      - value
+                    type: object
+                  propertyName:
+                    description: 'Required: The property name for which the desired/reported
+                      values are specified. This property should be present in the
+                      device model.'
+                    type: string
+                  reported:
+                    description: 'Required: the reported property value.'
+                    properties:
+                      metadata:
+                        description: Additional metadata like timestamp when the value
+                          was reported etc.
+                        type: object
+                      value:
+                        description: 'Required: The value for this property.'
+                        type: string
+                    required:
+                      - value
+                    type: object
+                required:
+                  - propertyName
+                type: object
+              type: array
+          type: object
+  version: v1alpha1
```

Comment:
Please update to v1alpha2, copy the yaml from build/crds

Reply:
> Please update to v1alpha2, copy the yaml from build/crds

Thanks for reminding. done